docker/develop/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ruby:2.4.4-slim-stretch
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update && \
+    apt-get install -y -qq \
+        build-essential \
+        cmake \
+        curl \
+        ghostscript \
+        git \
+        imagemagick \
+        libcurl4-openssl-dev \
+        libidn11-dev \
+        libmagickwand-dev \
+        libmariadbclient-dev \
+        libpq-dev \
+        libssl-dev \
+        libxml2-dev \
+        libxslt-dev \
+        nodejs \
+    && \
+    rm -rf /var/lib/apt/lists/*
+
+
+ARG DIA_UID
+ARG DIA_GID
+
+RUN addgroup --gid $DIA_GID diaspora && \
+    adduser \
+        --no-create-home \
+        --disabled-password \
+        --gecos "" \
+        --uid $DIA_UID \
+        --gid $DIA_GID \
+        diaspora \
+    && \
+    mkdir /home/diaspora /diaspora
+
+
+RUN curl -L -O \
+        https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 \
```

Comment:
You may use `http://cifiles.diasporafoundation.org/phantomjs-2.1.1-linux-x86_64.tar.bz2`, as we've had issues with randomly disappearing downloads in the past...

Reply:
We had issues with randomly disappearing cifiles too :stuck_out_tongue: but at least we can fix that ourselves :+1:

Reply:
... hey! I'm doing my best!!1